hat/conf/routes

```diff
@@ -29,3 +29,6 @@ POST        /control/v2/auth/passwordreset/confirm/:token        org.hatdex.hat.
 ->          /api/v2                                              v20.Routes
 ->          /api/v2.0                                            v20.Routes
 ->          /api/v2.6                                            v26.Routes
+
+# HAT Claim routes
+GET         /control/v2/claim                                    org.hatdex.hat.api.controllers.Authentication.claim(applicationId: String)
```

Comment:
ApplicationId is setup as a query parameter. If it actually should be query parameter (this endpoint actually should take a JSON body with applicationId and email attributes) then the type should be Option[String].

Reply:
Fixed.

Endpoint changed to a POST with body "applicationId" and "email"